Person/Person/main.m

```diff
@@ -51,13 +59,80 @@ - (NSString *)phoneNumber {
     return _phoneNumber;
 }
 
+
+- (BOOL)checkSameCity:(Person *)otherPerson{
+    
+    NSString *myCity = [self city];
+    
+    NSString *theirCity = [otherPerson city];
+    
+    if(self->_city == otherPerson->_city){
+        
+    }
+    
+    
+    if ([myCity isEqualToString: theirCity]){
+        NSLog(@"corect!");
+    } else {
+        NSLog(@"not correct!");
+    }
+    return YES;
```

Comment:
Your method is always returning YES! You should return the result of the `isEqualToString` method call...

```objective-c
   return [myCity isEqualToString: theirCity];
```